app/controllers/items/detail.js

```diff
@@ -132,6 +139,29 @@ export default GoodcityController.extend(
         .get("lastObject");
     }),
 
+    infoTabSelected: Ember.computed("tabName", function() {
+      return (
+        ["storage_detail", "storage_content", "info"].indexOf(
```

Comment:
Any reason for hard cording these values?

Reply:
Hardcoded values aside, I'm not sure I understand why the implementation isn't `this.get("tabName") === 'info'`
The info tab matches multiple tabs ? bit confused

Reply:
I have done this purposefully, because I want to select the info tab, when inner tabs are also selected. 

In case of box and pallets, be have nested tabs.

So I want to select info tab, when any of the nested tab is selected.

cc- @abulsayyad123 @Darshan-Chauhan @patrixr

Reply:
I have attached screen shot for reference.
![Screen Shot 2019-12-26 at 7 11 34 PM](https://user-images.githubusercontent.com/26408955/71478062-a838b600-2813-11ea-8011-c3ab4f13d65b.png)